swagger-spec/users/definition.yaml

```diff
@@ -64,6 +64,71 @@ properties:
         type: string
         readOnly: false
         description: 'The user''s timezone, e.g. ''Etc/UTC''.'
+      social:
+        type: object
+        readOnly: false
+        description: 'The social fields associated with a user.'
+        properties:
+          researcherId:
+            type: array
+            description: 'The researcherId for the given user.'
+            items:
+              type: string
+          baiduScholar:
+            description: 'The baiduScholar for the given user.'
+            type: array
+            items:
+              type: string
+          twitter:
+            description: 'The twitter for the given user.'
+            type: array
+            items:
+              type: string
+          ssrn:
+            description: 'The ssrn for the given user.'
+            type: array
+            items:
+              type: string
+          impactStory:
+            description: 'The impactStory for the given user.'
+            type: array
+            items:
+              type: string
+          researchGate:
```

Comment:
wow, I did not know we had these many social fields.  thanks for the docs 👍

Reply:
There are so many, right??? Never heard of so many of these services!